psd-web/spec/features/sign_in_spec.rb

```diff
@@ -21,13 +20,14 @@ def expect_incorrect_email_or_password
     expect(page).to have_css("h2#error-summary-title", text: "There is a problem")
     expect(page).to have_link("Enter correct email address and password", href: "#email")
     expect(page).to have_css("span#email-error", text: "Error: Enter correct email address and password")
+    expect(page).to have_css("span#password-error", text: "")
 
     expect(page).not_to have_link("Cases")
   end
 
-  context "when succeeeding signin in", :with_2fa do
+  context "when succeeding signin in", :with_2fa do
     context "when in two factor authentication page" do
-      it "allows user to sign in with correct two factor authentication code" do
+      scenario "allows user to sign in with correct two factor authentication code" do
```

Comment:
Found a mixture of `it` and `scenario` declarations through all this file. 
I defaulted everything to `scenario` as they're feature specs and wanted to keep a cohesive approach. They're not following the `feature/scenario` approach but is closer than before? Guess this file still needs some improvements.

Reply:
I like using `scenario` in feature specs, but if you rename them I think it’d be good to rename the description slightly too, so that it reads as a scenario, eg:

```suggestion
      scenario "user signing in with correct two factor authentication code" do
```

Reply:
Implemented it through all the file.